scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -781,42 +782,48 @@ public void setTaskDataOwnerIfNull() {
     }
 
     public void executeHousekeepingInDB(final List<Long> jobIdList, final boolean shouldRemoveFromDb) {
-        executeReadWriteTransaction(new HousekeepingSessionWork(jobIdList, shouldRemoveFromDb));
+        List<List<Long>> jobIdSubSets = Lists.partition(jobIdList, 1000);
```

Comment:
is it possible to make the '1000' a configurable param, or at least a varibale in the class

Reply:
Yes I willl look at this

Reply:
it is now configurable